addon/models/property.js

```diff
@@ -14,6 +14,14 @@ export default class Property {
     return this._property.type;
   }
 
+  get title() {
+    return this._property.title;
+  }
+
+  get default() {
+    return this._property.default;
```

Comment:
`default` refers to the default value of the property.  This is included in the JSON spec.